config/environments/production.rb

```diff
@@ -121,9 +121,7 @@
     allow do
       origins '*'
       resource '/packs/*', headers: :any, methods: [:get, :options]
-      if ActiveModel::Type::Boolean.new.cast(ENV.fetch('CW_API_ONLY_SERVER', false))
```

Comment:
Was this change intended?

Reply:
Oops, sorry! No, this is only for local testing purposes - will remove it in a second!